arcgis-ios-sdk-samples/Layers/Export tiles/ExportTilesViewController.swift

```diff
@@ -17,154 +17,242 @@ import UIKit
+        let areaOfInterest = frameToExtent()
+        // Get export parameters.
+        exportTask.exportTileCacheParameters(withAreaOfInterest: areaOfInterest, minScale: minScale, maxScale: maxScale) { [weak self, unowned exportTask] (params: AGSExportTileCacheParameters?, error: Error?) in
+            guard let self = self else { return }
+            if let params = params {
+                self.exportTiles(exportTask: exportTask, parameters: params, downloadFileURL: downloadFileURL)
+            } else if let error = error {
                 self.presentAlert(error: error)
-            } else if let params = params {
-                self.exportTilesUsingParameters(params)
             }
         }
     }
     
-    private func exportTilesUsingParameters(_ params: AGSExportTileCacheParameters) {
-        //destination path for the tpk, including name
-        let documentDirectoryURL = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first!
-        let downloadFileURL = documentDirectoryURL.appendingPathComponent("myTileCache.tpk")
-        
-        downloading = true
-        
-        //get the job
-        job = exportTask.exportTileCacheJob(with: params, downloadFileURL: downloadFileURL)
-        //run the job
+    /// Export tiles with the `AGSExportTileCacheJob` from the export task.
+    ///
+    /// - Parameters:
+    ///   - exportTask: An `AGSExportTileCacheTask` to run the export job.
+    ///   - parameters: The parameters of the export task.
+    ///   - downloadFileURL: A URL to where the tile package is saved.
+    func exportTiles(exportTask: AGSExportTileCacheTask, parameters: AGSExportTileCacheParameters, downloadFileURL: URL) {
+        // Get and run the job.
+        job = exportTask.exportTileCacheJob(with: parameters, downloadFileURL: downloadFileURL)
         job.start(statusHandler: { (status) in
-            //show job status
             SVProgressHUD.show(withStatus: status.statusString())
         }, completion: { [weak self] (result, error) in
-            //hide progress view
             SVProgressHUD.dismiss()
+            guard let self = self else { return }
             
-            guard let self = self else {
-                return
-            }
-
             self.job = nil
-            self.downloading = false
             
-            if let error = error {
-                if (error as NSError).code != NSUserCancelledError {
-                    self.presentAlert(error: error)
-                }
-            } else if let tileCache = result {
+            if let tileCache = result {
                 self.visualEffectView.isHidden = false
                 
                 let newTiledLayer = AGSArcGISTiledLayer(tileCache: tileCache)
                 self.previewMapView.map = AGSMap(basemap: AGSBasemap(baseLayer: newTiledLayer))
+                self.zoomToViewpoint(mapView: self.previewMapView, scale: 1e7)
```

Comment:
This line is added to cope with the new `.tpkx` package format. Without this line

- The original `.tpk` package loads correctly
- But the new `.tpkx` package does not show up at the correct extent

Comment out this line to see the difference. Is it expected?